### PR TITLE
Added scrolltop for result page navigation #15

### PR DESCRIPTION
**Pull request by masad-frost**
Took me longer than it should have to realize the scroll was on div-main.

Added animate because normal scrolling didn't feel natural. Though it's useless when the page doesn't quickly load or isn't preloaded, the throbber.

Edit: I can just use normal scrolltop if you think this affects the performance, I don't think it should though.

### CODE REVIEW

**Comment by amasad** [comment]
Good job, but you're repeating this same code in different locations, i.e. copypasta. [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself) it up!

File: lib/view.js [hunk 1]
```diff
@@ -61,10 +61,12 @@
       $olderPage.click(function () {
         currentPage++;
         $(historyModel).trigger('modelrefresh');
+        $('#div-main').animate({scrollTop:$('#div-main').position().top}, 'slow');
```
Comment (amasad) [inline lib/view.js]: space between key (`scrollTop:`) and value.

**Comment by amasad** [comment]
btw, you don't have to close this pull request to make changes, you can just push to your repo and this will update

**Comment by amasad** [comment]
will release tonight. Achievements unlocked for you:
1. contribute to opensource
2. contribute to app with more than ~10,000 monthly users that use it 100s of thousand times a month

:star2: :star: :stars: :100:

**Comment by masad-frost** [comment]
Great! 

Feels good. :smile: Wouldn't be a bad idea to have an achievement system for programmers, submit proof, reviewed by users, achievement acquired.

**Comment by amasad** [comment]
I think coderwall.com and maybe coderbit is trying to do that

**Pull request merged**